### PR TITLE
Directly dedent the spectral parameter docs.

### PR DESCRIPTION
**Pull request by anntzer**
There's little point in sending them through inspect.cleandoc.

Also list the defaults right next to the parameters.

(As discussed at length in https://github.com/matplotlib/matplotlib/pull/15765#discussion_r349900457 there are further improvements possible to these docstrings but let's not rehash that here.)

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/mlab.py [hunk 1]
```diff
@@ -613,72 +612,62 @@ def _single_spectrum_helper(x, mode, Fs=None, window=None, pad_to=None,
...
+    function is passed as the argument, it must take a data segment as an
+    argument and return the windowed version of the segment.
+
+sides : {'default', 'onesided', 'twosided'}
+    Which sides of the spectrum to return. 'default' is one-sided for real
+    data and two-sided for complex data. 'onesided' forces the return of a
+    one-sided spectrum, while 'twosided' forces two-sided.""",
+
+    Single_Spectrum="""\
+pad_to : int
```
Comment (timhoffm) [inline lib/matplotlib/mlab.py]:
```suggestion
pad_to : int, optional
```

File: lib/matplotlib/mlab.py [hunk 2]
```diff
@@ -613,72 +612,62 @@ def _single_spectrum_helper(x, mode, Fs=None, window=None, pad_to=None,
...
+    The number of points to which the data segment is padded when performing
+    the FFT.  While not increasing the actual resolution of the spectrum (the
+    minimum distance between resolvable peaks), this can give more points in
+    the plot, allowing for more detail. This corresponds to the *n* parameter
+    in the call to fft().  The default is None, which sets *pad_to* equal to
+    the length of the input signal (i.e. no padding).""",
+
+    PSD="""\
+pad_to : int
```
Comment (timhoffm) [inline lib/matplotlib/mlab.py]:
```suggestion
pad_to : int, optional
```

File: lib/matplotlib/mlab.py [hunk 3]
```diff
@@ -613,72 +612,62 @@ def _single_spectrum_helper(x, mode, Fs=None, window=None, pad_to=None,
...
+    the Fourier frequencies, *freqs*, in cycles per time unit.
+
+window : callable or ndarray, default: `.window_hanning`
+    A function or a vector of length *NFFT*.  To create window vectors see
+    `.window_hanning`, `.window_none`, `numpy.blackman`, `numpy.hamming`,
+    `numpy.bartlett`, `scipy.signal`, `scipy.signal.get_window`, etc.  If a
+    function is passed as the argument, it must take a data segment as an
+    argument and return the windowed version of the segment.
+
+sides : {'default', 'onesided', 'twosided'}
```
Comment (timhoffm) [inline lib/matplotlib/mlab.py]:
```suggestion
sides : {'default', 'onesided', 'twosided'}, optional
```

**Comment by anntzer** [comment]
thanks, all done

**Pull request merged**